backend/protos/mgmt/v1alpha1/job.proto

```diff
@@ -269,7 +269,39 @@ message MysqlTruncateTableConfig {
   bool truncate_before_insert = 1;
 }
 message MysqlOnConflictConfig {
+  // @deprecated - Use strategy nothing instead
   bool do_nothing = 1;
+
+  oneof strategy {
+    option (buf.validate.oneof).required = false;
+    // Do nothing when a conflict occurs
+    MysqlOnConflictDoNothing nothing = 2;
+    // Update columns when a conflict occurs
+    MysqlOnConflictUpdate update = 3;
+  }
+
+  // Do nothing strategy
+  message MysqlOnConflictDoNothing {}
+
+  // Update strategy
+  message MysqlOnConflictUpdate {
+    // // List of table-specific update configurations
+    // repeated MysqlOnConflictSchemaUpdate schemas = 1;
+  }
+
+  // message MysqlOnConflictSchemaUpdate {
```

Comment:
Assuming you are leaving these here with the intention of us eventually supporting it?

Reply:
correct